### PR TITLE
Create estimate API : - persister

### PR DESCRIPTION
**Pull request by pintu-eGov**

### CODE REVIEW

File: backend/estimate-service/src/main/java/org/egov/works/config/EstimateServiceConfiguration.java [hunk 1]
```diff
@@ -39,5 +39,12 @@ public void initialize() {
 
     @Value("${egov.mdms.search.endpoint}")
     private String mdmsEndPoint;
+
+    //Topic
+    @Value("${es.kafka.create.topic}")
```
Comment (rushang7-eGov) [inline backend/estimate-service/src/main/java/org/egov/works/config/EstimateServiceConfiguration.java:44]: What does es mean?

Comment (pintu-eGov) [reply backend/estimate-service/src/main/java/org/egov/works/config/EstimateServiceConfiguration.java:44]: Estimate Service

Comment (rushang7-eGov) [reply backend/estimate-service/src/main/java/org/egov/works/config/EstimateServiceConfiguration.java:44]: Instead es write estimate. ES generally means ElasticSearch.

Comment (pintu-eGov) [reply backend/estimate-service/src/main/java/org/egov/works/config/EstimateServiceConfiguration.java:44]: ok :)

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**